app/lib/associated_user_updater.rb

```diff
@@ -24,7 +24,7 @@ class AssociatedUserUpdater
   def initialize(params)
     @protocol_role = ProjectRole.find(params[:id])
     protocol = @protocol_role.protocol
-    eqm = EpicQueueManager.new(protocol, @protocol_role)
+    eqm = EpicQueueManager.new(protocol, params[:current_identity], @protocol_role)
```

Comment:
This new param will be missing from `app/controllers/associated_users_controller.rb`.

See `associated_users_controller.rb:84`
```updater = AssociatedUserUpdater.new(id: params[:id], project_role: project_role_params)```

Reply:
Fixed